src/parser/csql_grammar.y

```diff
@@ -28466,6 +28466,28 @@ pt_check_one_stmt(char* p)
                     }
                 }
           }
+        else if((p[0] == '-' && p[1] == '-') || (p[0] == '/' && p[1] == '/'))
+        {
+                for(p += 2; *p ; p++)
+                {
+                    if (*p == '\n')
+                    {
+                       p++;
+                       break;
+                    }
+                }
+        }
+        else if(p[0] == '/' && p[1] == '*')
+        {
+                for(p += 2; *p ; p++)
+                {
+                    if (p[0] == '*' && p[1] == '/')
+                    {
+                       p += 2;
+                       break;
+                    }
+                }
+        }
         else if(*p == ';')
           {
              p++;
```

Comment:
One of the following cases occurs when exiting the while statement.
1. If we change the first ';' to '\0'
2. When all characters are read because ';' is not encountered

If so, the next string must be ignored or is at the end of the string.

If so, wouldn't the processing below be unnecessary?
( Code below not included in this PR. )
```
while (*p == ' ' || *p == '\t' || *p == '\r' || *p == '\n')
  {
    p++;
  }
```

And is there some cases where this function returns false?